web/styles/portico/portico.css

```diff
@@ -1123,6 +1123,14 @@ input.new-organization-button {
             background-color: transparent;
         }
     }
+
+    #skip-navigation {
+        left: 80px !important;
```

Comment:
Let's refactor this so the `!important` keywords aren't necessary. I think you should be able to do that by running the non-media-query styles higher in the file, above the media queries.